Tests added

Cammelen opened the pull request:
Написаны следующие тесты:
1. Тесты на ручку "Создание пользователя"
2. Тесты на ручку "Логин пользователя"
3. Тесты на ручку "Изменение данных пользователя"
4. Тесты на ручку "Создание заказа"
5. Тесты на ручку "Получение заказов конкретного пользователя"

Commit:
Tests added

Commit:
Tests fixed

nikolay-977 commented on pom.xml:
```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Можно улучшить: хорошей практикой является добавление README.md с описанием проекта, которое включает название, технологии в проекте (например, Java 11, JUnit 4.13.2, maven 3.9.0 и т.д.), настройка (если необходима) и как запускать (mvn clean test). Как оформлять README.md можно ознакомиться в статье https://dzen.ru/a/YbzudflBPE1IDJF-

Cammelen replied:
Файл README.md был добавлен в "Tests added"
https://github.com/Cammelen/Diplom_2/commit/7f241be32cd5a574ac65699c7d73283cfc0184d0

nikolay-977 commented on src/test/java/BaseTest.java:
```diff
@@ -0,0 +1,49 @@
+    UserClient userClient = new UserClient();
+    OrdersClient ordersClient = new OrdersClient();
+
+    String bearerToken = "";
```
Необходимо исправить: В начале класса следует описывать переменные, а затем конструкторы и методы. Также можно установить курсор на корневую директорию и нажать сочетание клавиш Ctrl+Alt+L (Option+Command+L для Mac), затем выбрать checkbox rearrange entries и нажать ok.

Cammelen replied:
Исправлено в коммите "Second_commit"
https://github.com/Cammelen/Diplom_2/commit/328f3783ef8244df916454f4afd8995e7dba1f9a

nikolay-977 commented on src/test/java/BaseTest.java:
```diff
@@ -0,0 +1,49 @@
+
+    Random random = new Random();
+
+    String email = "user" + random.nextInt(100000) + "@yandex.ru";
```
Можно улучшить: Для всех полей следует указывать модификатор доступа.

Cammelen replied:
Исправлено в коммите "Second_commit"
https://github.com/Cammelen/Diplom_2/commit/328f3783ef8244df916454f4afd8995e7dba1f9a

nikolay-977 commented on src/test/java/ChangeUserTests.java:
```diff
@@ -0,0 +1,35 @@
+    }
+
+        @Test
+        @DisplayName("Изменение данных пользователя без авторизации с возвратом ошибки")
```
Можно улучшить: для расстановки пробелов и табуляции в соответствии с настройками Code Style в Idea используй сочетание клавиш Ctrl+Alt+L. Также можно установить курсор на корневую директорию и нажать клавиш Ctrl+Alt+L, отметить чек-боксы: cleanup code, а также optimize imports(для оптимизации импортов) и rearrange entries(для изменения порядка переменных и методов в соответствии с best practics)

Cammelen replied:
Исправлено в коммите "Second_commit"
https://github.com/Cammelen/Diplom_2/commit/328f3783ef8244df916454f4afd8995e7dba1f9a

nikolay-977 commented on src/test/java/LoginTests.java:
```diff
@@ -0,0 +1,36 @@
+    }
+}
+
+
```
Можно улучшить: лишних пропусков строк следует избегать. Для зрительного разделения кода достаточно использовать один пропуск строки. Пропуски строк должны быть однообразными.

Cammelen replied:
Исправлено в коммите "Second_commit"
https://github.com/Cammelen/Diplom_2/commit/328f3783ef8244df916454f4afd8995e7dba1f9a

Commit:
Second_commit

1. Исправлена структура в классах по порядку - переменные/конструкторы/методы
2. Для всех полей установлены модификаторы доступа
